src/main/java/training/CreateTable.java

```diff
@@ -0,0 +1,58 @@
+package training;
+
+import java.util.Arrays;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;  
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.dynamodbv2.model.AttributeDefinition;
+import com.amazonaws.services.dynamodbv2.model.KeySchemaElement;
+import com.amazonaws.services.dynamodbv2.model.KeyType;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType;
+
+public class CreateTable {
+
+    public static void main(String[] args) throws Exception {
+        System.out.println("Hii.");
+        try {
+        CreateTable createTable = new CreateTable();
+        AmazonDynamoDB client;
+        BasicAWSCredentials awsCreds = new BasicAWSCredentials("AKIASAQI2P2W4FOVXUMI", "5BenzeDNLgMlWwElnlPgNwr5Rh7E4fRdNWBsW0ei");

```

Comment:
Remove this code from here, we cannot check in this code as anyone can steal our credentials and use it to hack our aws account. Try to get it from environment variable.